manuscript/3-SoftConstr.txt

```diff
@@ -18,49 +18,48 @@
 ## Εισαγωγή
 
 
+
 Ο όρος κατασκευή λογισμικού αναφέρεται στη λεπτομερή κατασκευή ενός
```

Comment:
στη λεπτομερή κατασκευή ενός - detailed creation and maintenance of software. Δημιουργία και συντήρηση != κατασκευή. 
through==μέσω,όχι συνδυάζοντας
Δώσε έμφαση στην ακριβή απόδοση των λέξεων!